cpp/src/groupby/sort/sort_helper.cu

```diff
@@ -261,11 +261,11 @@ sort_groupby_helper::column_ptr sort_groupby_helper::sorted_values(
   column_view const& values, rmm::mr::device_memory_resource* mr, cudaStream_t stream)
 {
   column_ptr values_sort_order =
-    cudf::detail::sorted_order(table_view({unsorted_keys_labels(), values}),
-                               {},
-                               std::vector<null_order>(2, null_order::AFTER),
-                               mr,
-                               stream);
+    cudf::detail::stable_sorted_order(table_view({unsorted_keys_labels(), values}),
```

Comment:
This isn't really needed because the purpose of this method is to give you sorted values, not sorted keys. And they won't matter being stable or not.

Anyway, doesn't hurt to have it.